layersvt/device_simulation.cpp

```diff
@@ -4008,6 +4008,19 @@ VKAPI_ATTR VkResult VKAPI_CALL GetPhysicalDeviceSurfaceCapabilitiesKHR(VkPhysica
         VkSurfaceCapabilitiesKHR surf_caps = pdd->surface_capabilities_;
         surf_caps.currentExtent = pSurfaceCapabilities->currentExtent;
         surf_caps.currentTransform = pSurfaceCapabilities->currentTransform;
+
+        if (surf_caps.minImageCount == 0) surf_caps.minImageCount = pSurfaceCapabilities->minImageCount;
+        if (surf_caps.maxImageCount == 0) surf_caps.minImageCount = pSurfaceCapabilities->maxImageCount;
```

Comment:
`surf_caps.minImageCount` => `surf_caps.maxImageCount`

Reply:
Codegen could, but that is a later improvement.
This fixes the issue on my windows system.

Reply:
I'm okay with deferring the issue of whether to codegen.  I'll create an issue.

Reply:
https://github.com/LunarG/VulkanTools/issues/1564

I don't really know what I'm talking about, so if y'all could look at the issue and expand on it and correct any terminology I might have misused, I'd be appreciative. :-)